scss/themes/chat/dracula.scss

```diff
@@ -89,3 +89,13 @@ $genders: (
     --#{$varName}: #{$value};
   }
 }
+
+.user-bookmark,
+.message-event .user-bookmark {
+  color: $dracula-green !important;
```

Comment:
This doesn't have to be ``!important``. The theme should automatically override the base colours.